roles/splunk/defaults/main.yml

```diff
@@ -13,6 +13,7 @@ splunk_install_type: undefined # There are two ways to configure this. The easie
 splunk_install_path: /opt # Base directory on the operating system to which splunk should be installed
 splunk_nix_user: splunk
 splunk_nix_group: splunk
+splunk_licenses: undefined
```

Comment:
Wouldn't this make more sense to have a default value of `[]`?

Reply:
Yeah thanks, you're right. I added this to the newest pull request.